### ISSUE TITLE
Option disabledDates not found

### ISSUE THREAD

**Issue by danielspk** [Wikiki/bulma-calendar#240]
## Bug Report

**Environment**
- Bulma version: 6.0.7
- Browser: all
- How you are customizing the extension: no

**Current Behavior**

The disabledDates option does not disable dates.

**Input Code**

```js
const invalidDates = [
  new Date(new Date().getTime() -24 * 60 * 60 * 1000),
  new Date(new Date().getTime() + 192 * 60 * 60 * 1000)
]

const calendars = bulmaCalendar.attach('#myId[type="date"]', {
      lang: 'es',
      dateFormat: 'DD/MM/YYYY',
      displayMode: 'dialog',
      minDate: "2020-04-23",
      maxDate: "2020-08-19",
      showHeader: false,
      showFooter: false,
      disabledWeekDays: [0, 6],
      disabledDates: invalidDates
    })
```

The calendar initially shows all the dates, it does not respect the minimum or the maximum nor Saturdays and Sundays. If the `disabledDates` option is commented, the calendar works correctly but it is not possible to disable the required dates.

Is it a bug or a parameterization fault?

**Comment by danielspk** [comment]
This isn't a bug.

The option `dateFormat: 'DD/MM/YYYY'`  is responsible for the failure. Does anyone know how to set the dates to be disabled keeping the Latin format DD/MM/YYYY?

**Comment by mpeltomaa** [comment]
No idea, this date format thing is very confusing with this calendar. I haven't figured out how to keep disabledDates format stable. Locally, it works just fine, on development its totally different... 😬

**Comment by hsson** [comment]
Having problem with this too, a showstopper for sure.

**Comment by David-Ferrero1** [comment]
Hello, this my solution, and it's work:

`const options = {
      type: 'date',
      dateFormat: 'DD-MM-YYYY',
      displayMode: 'dialog',
      lang: 'fr',
      disabledDates: invalidDates,
    };`

**Comment by michael-hack** [comment]
Should be fixed with 6.1.5
